src/period.cpp

```diff
@@ -8,6 +8,15 @@
 #include "nanotime/pseudovector.hpp"
 #include "nanotime/utilities.hpp"
 
+// See comment in <R_Ext/Complex.h> in R>=4.3.0 and #134. Old
+//   approach to do `Rcomplex{ re, im }` now gives compiler issue
+//   on some compilers, R versions.
+inline Rcomplex makeComplex(double re, double im) {
```

Comment:
I switched this to `(double re, double im)`. I _think_ that matches the previous behavior better, as `Rcomplex{re,im}` would create the `Rcomplex` object taking copies of `re`, `im`?